### PR TITLE
Updating Member List for Year 2016-17

### PR DESCRIPTION
**Pull request by ujdcodr**

### CODE REVIEW

File: people.html [hunk 1]
```diff
@@ -53,80 +55,106 @@
...
-              <li>JAYASOORYAN KV</li>
-              <li>Vaibhav Salve</li>
-              <li>Aadheeshwar Vijayakumar</li>
-              <li><a href="https://therealkbhat.github.io">Kishor Bhat</a></li>
-              <li>akshay sharma</li>
-              <li>Grama Pramod Anirudh</li>
-              <li>Vadiraja K</li>
               <li><a href="https://adeepkit01.github.io">Ankit Deepak</a></li>
-              <li>Shravya K.S</li>
+              <li><a href="https://sigsevneo.github.io">Anagh Singh</a></li>
```
Comment (sigsevneo) [inline people.html]: The alphabetical nature seems to be off here. Reorder entries for Ankit and I to be below Akshay.

File: people.html [hunk 2]
```diff
@@ -53,80 +55,106 @@
...
               <li>Mukta Kulkarni</li>
               <li>Neha B</li>
-              <li>Param Hanji</li>
+              <li>Pidaparthi Lakshmi Aparna</li>
               <li>Pooja Mahadev Soundalgekar</li>
-              <li>Prabhanjan Mannari</li>
               <li>Prajwala T M</li>
-              <li>Rohan Chetan Thanki</li>
+              <li>Rohit Sharma</li>
+              <li>Vignesh K</li>
```
Comment (sigsevneo) [inline people.html]: V after T.

File: people.html [hunk 3]
```diff
@@ -53,80 +55,106 @@
...
+              <li>Arvind kumar</li>
+              <li>Barun Kumar Acharya</li> 
+              <li>Bhabesh Chanduka</li>
+              <li>Derik Clive R</li>
+              <li>G Leela Siva Sai Kumar</li>
+              <li>Hafeez Ali</li>
+              <li>Hrishikesh Hiraskar</li>
+              <li>Lokaksha BM</li>
+              <li>M M Vikram</li>
+              <li>V K Prakrith Abhishekk</li>
```
Comment (sigsevneo) [inline people.html]: V after U.

File: people.html [hunk 4]
```diff
@@ -53,80 +55,106 @@
...
+              <li>Sagar Bharadwaj KS</li>
+              <li>Sahil Sharma</li>
+              <li>Shah Het J</li>
+              <li>Shivananda D</li>
+              <li>Siddharth Gangwani</li>
+              <li>Somnath Sarkar</li>
+              <li>Ujjwal Pasupulety</li>
+              <li>Vilas M</li>
+              <li>Zoeb Mithaiwala</li>
+              <li>Kapil Bharath</li>
```
Comment (sigsevneo) [inline people.html]: K before L.

**Review by sigsevneo — CHANGES_REQUESTED** [review]

**Comment by ujdcodr** [comment]
Anything else?

**Review by sigsevneo — APPROVED** [review]

**Comment by ujdcodr** [comment]
Yaayyy! Thanks :-P

**Pull request merged**